js/functions.js

```diff
@@ -133,22 +130,230 @@ let trignoOperations = [
+  } else if (clickedItem.includes("cos")) {
+    cosTrignoOperations(clickedItem, value);
+  } else if (clickedItem.includes("tan")) {
+    tanTrignoOperations(clickedItem, value);
+  } else if (clickedItem.includes("csc")) {
+    cscTrignoOperations(clickedItem, value);
+  } else if (clickedItem.includes("sec")) {
+    secTrignoOperations(clickedItem, value);
+  } else if (clickedItem.includes("cot")) {
+    cotTrignoOperations(clickedItem, value);
+  }
+}
+
+// let combinedFn = {
+//   "sin":Math.sin(),
+//   "cos":Math.cos(),
+// }
+
+//to handle all the sin operations
+function sinTrignoOperations(clickedItem, value) {
+  switch (clickedItem) {
+    case "sin":
+      value = Math.sin(value);
+      break;
+    case "sin-in":
+      if (value <= 1 && value >= -1) {
+        value = Math.asin(value);
+        setCharAtInputField(value);
+      } else {
+        showErrForSomeTime("Please enter the input from -1 to 1 (RAD)");
+        return;
+      }
+    case "sin-h":
+      value = Math.sinh(value);
+      break;
+    case "sin-h-in":
+      value = Math.asinh(value);
+      break;
+  }
+  if (isNaN(value)) {
+    showErrForSomeTime();
+    return;
+  }
+  setCharAtInputField(value);
+}
+
+//to handle all the cos operations
+function cosTrignoOperations(clickedItem, value) {
+  switch (clickedItem) {
+    case "cos":
+      value = Math.cos(value);
+      break;
+    case "cos-in":
+      if (value <= 1 && value >= -1) {
+        value = Math.acos(value);
+        setCharAtInputField(value);
+      } else {
+        showErrForSomeTime("Please enter the input from -1 to 1 (RAD)");
+        return;
+      }
+      break;
+    case "cos-h":
+      value = Math.cosh(value);
+      break;
+    case "cos-h-in":
+      value = Math.acosh(value);
+      break;
+  }
+  if (isNaN(value)) {
+    showErrForSomeTime();
+    return;
+  }
+  setCharAtInputField(value);
+}
+
+//to handle all the tan operations
+function tanTrignoOperations(clickedItem, value) {
+  switch (clickedItem) {
+    case "tan":
+      value = Math.tan(value);
+      break;
+    case "tan-in":
+      value = Math.atan(value);
+      break;
+    case "tan-h":
+      value = Math.tanh(value);
+      break;
+    case "tan-h-in":
+      if (value <= 1 && value >= -1) {
+        value = Math.atanh(value);
+      } else {
+        showErrForSomeTime("Please enter the input from -1 to 1 (RAD)");
+        return;
+      }
+      break;
+  }
+  if (isNaN(value)) {
+    showErrForSomeTime();
+    return;
+  }
+  setCharAtInputField(value);
+}
+
+//to handle all the cosec operation
+function cscTrignoOperations(clickedItem, value) {
+  switch (clickedItem) {
+    case "csc":
+      value = 1 / Math.sin(value);
+      break;
+    case "csc-in":
+      if (value <= 1 && value >= -1) {
+        value = 1 / Math.asin(value);
+        setCharAtInputField(value);
+      } else {
+        showErrForSomeTime("Please enter the input from -1 to 1 (RAD)");
+        return;
+      }
```

Comment:
`return` is defined in else block.
Please confirm, execution can move to next case since no break / return is present

Reply:
added the break to handle the switch case condition.